README.md

```diff
@@ -62,6 +62,34 @@ curl -X POST http://localhost:3001/reset/{token} \
   -d '{ "password": "mynewpassword" }'
 ```
 
+#### Google Sign in
+
+POST /provider/google
+
+```bash
+curl -X POST \
+  http://localhost:3001/provider/google \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"token": "<google access token>",
+	"state":"<state defined in config>"
```

Comment:
What's the goal of the state ? What config are you talking about?

Reply:
in config.go / environment variable 
state : response up